pkg/cmd/project/item-edit/item_edit.go

```diff
@@ -63,17 +63,18 @@ func NewCmdEditItem(f *cmdutil.Factory, runF func(config editItemConfig) error)
 		Short: "Edit an item in a project",
 		Long: heredoc.Docf(`
 			Edit either a draft issue or a project item. Both usages require the ID of the item to edit.
-			
-			For non-draft issues, the ID of the project is also required, and only a single field value can be updated per invocation.
+
+			For non-draft issues, the ID of the project is also required,
```

Comment:
The thing I don't understand just looking at the changes is how you're determining some of these lines should be wrapped versus not.  This is something I'd feel more comfortable with maintainers having the decision deciding with insight from contributors.  To note, the line above isn't wrapped, so I'm unsure how important this is to address here and how we'd do it if we decided to.